### PR TITLE
Modifications to OC-Accel to enable more functionality of 9H3 board and help in HDL action development

### PR DESCRIPTION
**Pull request by fleon-psi**

### CODE REVIEW

File: hardware/hdl/core/snap_global_vars.v_source [hunk 1]
```diff
@@ -83,6 +83,14 @@
...
 #endif
 
+#if defined(CONFIG_ENABLE_9H3_LED)
+`define ENABLE_9H3_LED
+#endif
+
+#if defined(CONFIG_ENABLE_9H3_EEPROM)
+`define ENABLE_9H3_EEPROM
+#endif
+
```
Comment (bmesnet) [inline hardware/hdl/core/snap_global_vars.v_source:93]: Isn't CONFIG_ENABLE_9H3_AVR missing here ?

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +1,3 @@
 [submodule "hardware/oc-bip"]
 	path = hardware/oc-bip
-	url = https://github.com/OpenCAPI/OpenCAPI3.0_Client_RefDesign.git
+	url = https://github.com/opencapi/OpenCAPI3.0_Client_RefDesign.git
```
Comment (bmesnet) [inline .gitmodules:3]: Seems there is case typo on opencapi word in the link!

**Review by acastellane — COMMENTED** [review]
Changes do not break any test so far. I'm good to go. Thanks for updates

**Review by acastellane — APPROVED** [review]

File: hardware/hdl/core/snap_global_vars.v_source (hunk 1 above)
Comment (acastellane) [reply hardware/hdl/core/snap_global_vars.v_source:93]: Agreed, corrected

**Review by bmesnet — CHANGES_REQUESTED** [review]
.gitmodules needs to be corrected

**Review by bmesnet — APPROVED** [review]
seems the .gitmodules case typo isn't breaking anything - sorry for this warning

**Pull request merged**